ios/RNCallKeep/RNCallKeep.m

```diff
@@ -54,8 +55,10 @@ - (instancetype)init
     NSLog(@"[RNCallKeep][init]");
 #endif
     if (self = [super init]) {
-        _isStartCallActionEventListenerAdded = NO;
-        _delayedEvents = [NSMutableArray array];
+        if (_delayedEvents == nil) {
```

Comment:
init gets called several times, each time nuking this collection, resulting in us not getting any events at all.